java-grpc/flatbuffers-compiler/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
Why can't this be under `grpc/` like everything else? Also, it seems like this is conflating 2 issues: making `flatc` available for Java users, and support for gRPC in Java, ideally those would be independent, and independent PRs. This makes it harder to review. It is hard for me to see if using `flatc` in Java without gRPC is possible, for example.